source/fuzz/fuzzer_pass_add_parameters.cpp

```diff
@@ -91,5 +151,91 @@ uint32_t FuzzerPassAddParameters::GetNumberOfParameters(
   return static_cast<uint32_t>(type->AsFunction()->param_types().size());
 }
 
+uint32_t FuzzerPassAddParameters::FindOrCreateLocalVariable(
+    uint32_t pointer_type_id, uint32_t function_id,
+    bool pointee_value_is_irrelevant) {
+  uint32_t result_id = 0;
+  auto pointer_type = GetIRContext()->get_type_mgr()->GetType(pointer_type_id);
+  // No unused variables in release mode.
+  (void)pointer_type;
+  assert(pointer_type->AsPointer() &&
```

Comment:
You can use `assert(pointer_type && pointer_type->AsPointer() && pointer_type->AsPointer()->storage_class() == Function...`

Reply:
i.e. don't forget to check that `pointer_type` is not null.

Reply:
However, maybe it's OK if this function is only used by the `FuzzerPassAddParameters`. @antnkarp, @paulthomson, are we planning on adding it to the `FuzzerPass`?